Logarlec/src/modul/HolyBeerCup.java

```diff
@@ -158,6 +158,9 @@ public void SetDurability(int durability) {
 	public boolean PickedUpStudent(Student st) {
 		Logger.started(this, "PickedUpStudent", st);
 		boolean isAdded = st.AddToInventory(this);
+		if(isAdded && effectDuration > 0){
+			st.AddHolyBeerCup(this);
```

Comment:
Ugy emlekszem ugy beszeltuk h a holyBeerCupot is aktivalni kell, szoval csak activatenel kell belerakni a listajaba